pkg/yang/types_builtin.go

```diff
@@ -450,12 +451,14 @@ var (
 )
 
 const (
-	MaxInt64        = 1<<63 - 1 // maximum value of a signed int64
-	MinInt64        = -1 << 63  // minimum value of a signed int64
-	AbsMinInt64     = 1 << 63   // the absolute value of MinInt64
-	MaxEnum         = 1<<31 - 1 // maximum value of an enum
-	MinEnum         = -1 << 31  // minimum value of an enum
-	MaxBitfieldSize = 1 << 32   // maximum number of bits in a bitfield
+	MaxInt64                = 1<<63 - 1        // maximum value of a signed int64
+	MinInt64                = -1 << 63         // minimum value of a signed int64
+	AbsMinInt64             = 1 << 63          // the absolute value of MinInt64
+	MaxEnum                 = 1<<31 - 1        // maximum value of an enum
+	MinEnum                 = -1 << 31         // minimum value of an enum
+	MaxBitfieldSize         = 1 << 32          // maximum number of bits in a bitfield
+	MaxFractionDigits uint8 = 18               // maximum fractional digits per Section 9.3.4
+	MaxFloat64              = float64(1 << 48) // maximum absolute float64 that can be converted to decimal
```

Comment:
Just pulled that out of the air :-) I didn't want to assume 53 bits but something more conservative. But this is a moot point since I'm getting rid of it altogether.